e2e/tests-dfx/assetscanister.bash

```diff
@@ -52,6 +52,100 @@ check_permission_failure() {
   assert_contains "Caller does not have Prepare permission"
 }
 
+@test "deploy --by-proposal happy path" {
```

Comment:
would it makes sense to also add tests for the sad path?

Reply:
sure would